scripts/test/Muon/loadfile_view_test.py

```diff
@@ -0,0 +1,48 @@
+import unittest
```

Comment:
if the view is going to have logic, we should test it in here

Reply:
Yes, so that logic you mentioned in the previous comments is actually tested here.

Reply:
ah that was not obvious to me. I will have another look